### DISCUSSION TITLE
Impossible to install Brew

### DISCUSSION THREAD

**Discussion by djoo** [Homebrew/discussions#1686] [Getting started]
Hi,
I try to install Brew but it doesn't works.
I have this error :

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password).
Password:
==> This script will install:
/opt/homebrew/bin/brew
/opt/homebrew/share/doc/homebrew
/opt/homebrew/share/man/man1/brew.1
/opt/homebrew/share/zsh/site-functions/_brew
/opt/homebrew/etc/bash_completion.d/brew
/opt/homebrew

Press RETURN to continue or any other key to abort
==> /usr/bin/sudo /usr/sbin/chown -R johannroche:admin /opt/homebrew
==> Downloading and installing Homebrew...
HEAD is now at d3887df2a Merge pull request #11529 from Homebrew/sorbet-files-update
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

What can i do ? Thanks

**Comment by carlocab** [comment]
See #666 for a discussion of related problems/solutions.

**Reply by djoo** [reply]
Thanks a lot
